src/java/org/apache/cassandra/db/SerializationHeader.java

```diff
@@ -489,7 +489,7 @@ public SerializationHeader toHeader(String descriptor, TableMetadata metadata, V
                         // deserialization. The column will be ignored later on anyway.
                         column = metadata.getDroppedColumn(name, isStatic);
                         if (column == null)
-                            throw new UnknownColumnException("Unknown column " + UTF8Type.instance.getString(name) + " during deserialization");
+                            throw new UnknownColumnException("Unknown column " + UTF8Type.instance.getString(name) + " during deserialization of " + metadata.keyspace + '.' + metadata.name);
```

Comment:
Yes yes and yes. Otherwise on escalations it's impossible to follow what's going on.